src/connector/BlendWithdrawActions.ts

```diff
@@ -25,11 +25,29 @@ export async function withdraw(
   const amount0Min = estimated0.mul(1 - ratioChange / 100);
   const amount1Min = estimated1.mul(1 - ratioChange / 100);
 
+  let transactionOptions: any = {};
+
+  try {
+    const estimatedGas = (
+      (await tokenContract.estimateGas.withdraw(
+        shares.toFixed(0),
+        amount0Min.toFixed(0),
+        amount1Min.toFixed(0)
+      )) as BigNumber
+    ).toNumber();
+
+    transactionOptions['gasLimit'] = (estimatedGas * 1.1).toFixed(0);
```

Comment:
Should `1.1` be a constant shared between BlendWithdrawActions and BlendDepositActions?

Reply:
Yes, it should be. Updating...